contracts/InterchainTokenFactory.sol

```diff
@@ -121,7 +121,7 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
      * @param symbol The symbol of the token.
      * @param decimals The number of decimals for the token.
      * @param initialSupply The amount of tokens to mint initially (can be zero).
```

Comment:
Mention that initial supply goes to the sender of this method (and not the minter address).